pymchelper/writers/plots.py

```diff
@@ -172,14 +193,21 @@ def write(self, detector):
         plt.xlabel(self._make_label(detector.units[x_axis_number], x_axis_name))
         xlist = list(detector.axis_values(0, plotting_order=True))  # make list of values from generator
 
+        # configure logscale on X and Y axis (both for positive and negative numbers)
+        if PlotAxis.x in self.axis_with_logscale:
+            plt.xscale('symlog')
+
+        if PlotAxis.y in self.axis_with_logscale:
+            plt.yscale('symlog')
+
         # 1-D plotting
         if detector.dimension == 1:
 
             # add optional error area
             if np.any(detector.error):
                 plt.fill_between(xlist,
                                  (data - error).clip(0.0),
-                                 (data + error).clip(0.0, 1.05 * (detector.v.max())),
+                                 (data + error).clip(0.0, 1.05 * data.max()),
```

Comment:
Out of curiosity - is this numpy.clip or something else?

Reply:
Yes, its numpy clip: both `data` and `error` are numpy arrays

Reply:
numpy rulez!